src/screens/Feed.js

```diff
@@ -146,3 +116,73 @@ const styles = StyleSheet.create({
     justifyContent: 'space-between',
   },
 });
+
+
+const posts = [
+  {
+    id: 0,
+    img:
+      'https://www.familyandmedia.eu/wp-content/uploads/2018/10/529382-4386816-selfie-psicologia-725x545.jpg',
+    caption:
+      'Quarantine selfie oronasa dsajl as lnsaln aln flsan lsan lfaslf fnasl nfals nlsan lsan lasn lfasn lfasnf lan',
+    likes: 2,
+    saves: 0,
+    comments: [
+      {
+        userName: 'Model1234',
+        userImage:
+          'https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
+        comment: 'Sla girlll!!',
+      },
+    ],
+    username: 'gabriela',
+    likers: {
+      dennyliang: true,
+    },
+    bookmarkers: {
+      dennyliang: false,
+    },
+  },
+  {
+    id: 1,
+    img:
+      'https://www.familyandmedia.eu/wp-content/uploads/2018/10/529382-4386816-selfie-psicologia-725x545.jpg',
+    caption: 'Quarae selfie #corona',
+    likes: 2,
+    saves: 0,
+    comments: [
+      {
+        userName: 'Model1234',
+        userImage:
+          'https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
+        comment: 'Slay girl!!',
+      },
+    ],
+    username: 'gabriellllla',
+    likers: {
+      dennyliang: false,
+    },
+    bookmarkers: {},
+  },
+  {
+    id: 2,
+    img:
+      'https://www.familyandmedia.eu/wp-content/uploads/2018/10/529382-4386816-selfie-psicologia-725x545.jpg',
+    caption: 'Quarantine selfie #corona',
+    likes: 2,
+    saves: 0,
+    comments: [
+      {
+        userName: 'Model1234',
+        userImage:
+          'https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
+        comment: 'Slay girllll!!!',
+      },
+    ],
+    username: 'gabriela',
+    likers: {},
+    bookmarkers: {
+      dennyliang: true,
+    },
+  },
+];
```

Comment:
Consider either moving these plug-in/default values to the database, or automating the process by writing a function and avoiding repetition